apps/web/src/modules/job/list/components/FilterContainer/index.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+  AccessTime,
+  ArrowDropDownOutlined,
+  DateRangeOutlined,
+} from '@mui/icons-material'
+import {
+  Checkbox,
+  Divider,
+  FormControlLabel,
+  FormGroup,
+  Typography,
+} from '@mui/material'
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs'
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider'
+import { MobileDatePicker } from '@mui/x-date-pickers/MobileDatePicker'
+import { MobileTimePicker } from '@mui/x-date-pickers/MobileTimePicker'
+import TextField from 'common/components/TextField'
+import { Dayjs } from 'dayjs'
+import React from 'react'
+
+import { FilterBox } from './style'
+
+export default function SearchBox() {
+  const [startShooting, setStartShooting] = React.useState<Dayjs | null>(null)
+  const [endShooting, setEndShooting] = React.useState<Dayjs | null>(null)
+  const [date, setDate] = React.useState<Dayjs | null>(null)
+  return (
+    <FilterBox>
+      <Typography variant="body1"> การถ่ายทำ </Typography>
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
```

Comment:
Is this necessary? (I already include this it `_app.tsx` file )

Reply:
K forgot to read _app.tsx lol